### PR TITLE
EPMRPP-74453 || Leave social media icons only in footer

### PR DESCRIPTION
**Pull request by Vadim73i**

### CODE REVIEW

File: src/views/components/footer/Footer.scss [hunk 1]
```diff
@@ -58,6 +58,23 @@
...
       margin-left: 17px;
+
+      &.git {
+        margin-right: 9px;
+        cursor: pointer;
+        overflow: hidden;
+        width: 100px;
+
+        .count-block {
+          margin-left: 5px;
```
Comment (chivekrodis) [inline src/views/components/footer/Footer.scss:69]: Please add white background for the github tooltip

**Review by AmsterGet — CHANGES_REQUESTED** [review]
The menu items should be centered (see the screenshot below).

image

Page should be called **Pricing**.

Also please add a pricing menu item to mobile view.

image

File: src/views/components/header/Header.scss [hunk 2]
```diff
@@ -280,29 +269,6 @@
             box-sizing: border-box;
             cursor: pointer;
 
-            &.git {
-              width: 100px;
-              padding: 15px 13px;
-            }
-            &.slack {
-              background: url(../../../common/img/social/icon-sl.svg) center no-repeat;
-            }
```
Comment (chivekrodis) [inline src/views/components/header/Header.scss]: It doesn't seem to need to be removed.. Please recheck mobile view, we have side panel

File: src/views/components/header/index.js [hunk 3]
```diff
@@ -24,12 +21,6 @@ export default Epoxy.View.extend({
   initialize(options) {
     this.mainScrollEl = options.mainScrollEl;
     this.renderTemplate();
-    this.headerSocial = new HeaderSocial();
-    $('[data-js-social-container]', this.$el).html(this.headerSocial.$el);
-    this.gitHubStarsCount = new GitHubStarsCount();
-    $('[data-js-github-stars-container]', this.$el).html(this.gitHubStarsCount.$el);
```
Comment (chivekrodis) [inline src/views/components/header/index.js]: `data-js-github-stars-container` necessary container for side panel

**Review by chivekrodis — CHANGES_REQUESTED** [review]

File: src/views/components/footer/Footer.scss [hunk 4]
```diff
@@ -67,6 +67,7 @@
 
         .count-block {
           margin-left: 5px;
+          background-color: #ffffff;
```
Comment (chivekrodis) [inline src/views/components/footer/Footer.scss]: Please replace it with variable `$COLOR-WHITE_TWO`

**Review by chivekrodis — CHANGES_REQUESTED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**